R/license.R

```diff
@@ -55,8 +55,14 @@ use_gpl3_license <- function(name = find_name()) {
   force(name)
   check_is_package("use_gpl3_license()")
 
-  use_description_field("License", "GPL-3", overwrite = TRUE)
+  use_description_field("License", "GPL-3 + file LICENSE", overwrite = TRUE)
   use_license_template("GPL-3", name)
+
+  use_template(
+    "license-mit.txt",
```

Comment:
This is confusing (although it produces the correct result). I think we should rename this file, so the code is less puzzling.

I propose `year-copyright.txt`. After you rename it, you'll have to update the MIT license code to use the new name too.

Reply:
It should be good :)